configure

```diff
@@ -1441,6 +1447,8 @@ Optional Features:
   --enable-verifyheap     Enable advanced heap verification
   --enable-devmode        Developper mode: turn on warnings and debugging
```

Comment:
*Developer mode

Reply:
Thanks for catching these typos celtic-coder!

Reply:
You are welcome! 🙂